config/initializers/new_framework_defaults.rb

```diff
@@ -20,4 +20,4 @@
 # Rails.application.config.active_record.belongs_to_required_by_default = false
 
 # Do not halt callback chains when a callback returns false. Previous versions had true.
-ActiveSupport.halt_callback_chains_on_return_false = true
+ActiveSupport.halt_callback_chains_on_return_false = false
```

Comment:
So I'm PRETTY sure this was a mongoid thing but I can't remember. Definitely something to check on but I'm pretty sure this is okay.